README.md

```diff
@@ -3,34 +3,35 @@
 # Lab work <mark>NUMBER</mark>: <mark>SHORT TOPIC</mark>
 Authors (team): <mark>AUTHORS WITH GITHUB LINKS</mark><br>
 Variant: <mark>VARIANT SHOULD BE HERE</mark>
-## Before 
+## Prerequisites
 
-<mark>DESCRIBE PREREQUIREMENTS</mark>
+<mark>DESCRIBE PREREQUIREMENTS AS GCC OPENMP WHATEVER</mark>
 
-### Compiling
+### Compilation
 
-<mark>DESCRIBE PREREQUIREMENTS AND INSTALL HERE</mark>
+<mark>HOW SHOULD YOUR PROGRAM BE COMPILED (./comile.sh RECOMMENDED)</mark>
 
-### Installing
+### Installation
 
-<mark>DESCRIBE THE INSTALL HERE</mark>
+<mark>DESCRIBE THE INSTALL PROCCES HERE (USE ./dependencies FOLDER)</mark>
 
 ### Usage
 
-<mark>DESCRIBE THE USAGE HERE</mark>
+<mark>PROVIDE EXAMPLE HOW TO RUN YOUR PROGRAM HERE</mark>
 
 ### Important!
 
-<mark>WHAT ELSE WE SHOULD KNOW ABOUT YOUR WORK</mark>
+<mark>WHAT ELSE WE SHOULD KNOW ABOUT YOUR WORK?</mark>

```

Comment:
WHAT ELSE WE SHOULD KNOW ABOUT YOUR WORK? (E.G. KNOWN ISSUES, BUGS, SPECIAL BEHAVIOR ETC)